azure-pipelines/dotnet.yml

```
@@ -10,9 +10,9 @@ steps:
   displayName: 🧪 dotnet test
   condition: and(and(succeeded(), ${{ parameters.RunTests }}), ne(variables['BuildConfiguration'], 'ReleaseMac'))
 
-# - script: dotnet build ./test/Xunit.StaFact.Tests.Mac/Xunit.StaFact.Tests.Mac.csproj /t:Test
-# displayName: 🧪 Mac-specific tests
-#  Condition: eq(variables['BuildConfiguration'], 'ReleaseMac')
+ - script: dotnet build ./test/Xunit.StaFact.Tests.Mac/Xunit.StaFact.Tests.Mac.csproj /t:Test
+ displayName: 🧪 Mac-specific tests
+  Condition: eq(variables['BuildConfiguration'], 'ReleaseMac')
```

Fix the indentation and the capitalization.

```suggestion
- script: dotnet build ./test/Xunit.StaFact.Tests.Mac/Xunit.StaFact.Tests.Mac.csproj /t:Test
  displayName: 🧪 Mac-specific tests
  condition: eq(variables['BuildConfiguration'], 'ReleaseMac')
```